app/components/Application/ApplicationOverrideJustification.tsx

```diff
@@ -110,79 +114,83 @@ export const ApplicationOverrideJustificationComponent: React.FunctionComponent<
     );
   }
 
-  return (
-    <>
-      <Alert variant="danger">
-        <Accordion>
-          <div className="override-accordion">
-            Your application contains errors shown below that must be fixed
-            before submission. You may either correct these or alternatively,
-            override and provide justification. Your justification will be
-            reviewed by CAS and may cause a delay in processing your
-            application.
-            <Accordion.Toggle
-              as={Button}
-              variant="secondary"
-              eventKey="0"
-              onClick={() => setAccordionOpen(!accordionOpen)}
-            >
-              Override and Justify
-              <FontAwesomeIcon
-                icon={faCaretDown}
-                style={{marginLeft: '0.75em'}}
-              />
-            </Accordion.Toggle>
-          </div>
-          <Accordion.Collapse in={accordionOpen} eventKey="0">
-            <>
-              <Form>
-                <h4>Override Form Validation</h4>
-                <Form.Group controlId="overrideJustification">
-                  <Form.Label>Justification for incomplete form:</Form.Label>
-                  <Form.Control
-                    as="textarea"
-                    rows={4}
-                    value={overrideJustification || ''}
-                    onChange={handleOverrideChange}
-                  />
-                </Form.Group>
-                <Button variant="success" onClick={handleOverrideSave}>
-                  Save
-                </Button>
-                <Accordion.Toggle
-                  as={Button}
-                  eventKey="0"
-                  variant="light"
-                  style={{
-                    marginLeft: '1em',
-                    border: '1px solid currentColor'
-                  }}
-                  onClick={handleOverrideCancel}
-                >
-                  Cancel
-                </Accordion.Toggle>
-              </Form>
-              {!overrideJustification && (
-                <p className="justification-text">{emptyError}</p>
-              )}
-            </>
-          </Accordion.Collapse>
-        </Accordion>
-      </Alert>
-      <style jsx>
-        {`
-          .override-accordion {
-            display: flex;
-            flex-wrap: wrap;
-            justify-content: flex-end;
-          }
-          .justification-text {
-            color: red;
-          }
-        `}
-      </style>
-    </>
-  );
+  if (hasErrors) {
```

Comment:
instead of having this big block in an `if` you shoud use a [guard clause](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) returning `null`